### ISSUE TITLE
Allow editing and saving of files like sane image editors

### ISSUE THREAD

**Issue by sketchbuch** [glimpse-editor/Glimpse#117]
If I edit a jpg I want to be able to hit save or ctrl + s and just save it. I don't want some wierd native gimp format, have to select overwrite and then click through several dialog windows of file settings.

**Comment by Member1221** [comment]
Imo the best way of approaching that would be to merge the features of exporting and saving. XCF should just be a save format together with all the other file formats. XCF should be the default format, but it should be easy to save png files, etc. and when opened save should save as the opened file's type.

**Comment by pipboy96** [comment]
I agree with @Member1221, this is also what Photoshop, Paint.NET and virtually every image editor in existence is doing.

**Comment by TrechNex** [comment]
Linking to #394 as this is similar functionality, albeit in the form of a keyboard shortcut
